test/docker/container.py

```diff
@@ -38,6 +37,30 @@ def container(request, base_name: str, image: Union[Image, str], start: bool = T
         if start:
             container.start()
         yield container
+    finally:
+        # int(f"\nRemoving container {container_name}")
+        client.containers.get(container_name).remove(force=True)
+        client.close()
+
+
+def timestamp() -> str:
+    return f'{datetime.now().timestamp():.0f}'
+
+
+@contextmanager
+def container_context(image_name: str, suffix: str = None, start: bool = True, **kwargs):
+    container_name = sanitize_container_name(f"{image_name}_{suffix or timestamp()}")
+    client = docker.from_env()
+    try:
+        container = client.containers.create(
+            image=image_name,
+            name=container_name,
+            detach=True,
+            **kwargs
+        )
+        if start:
+            container.start()
+        yield container
     finally:
         client.containers.get(container_name).remove(force=True)
         client.close()
```

Comment:
duplication

Reply:
Thanks for this finding.
See latest push removing the duplicated code.